pkgs/development/libraries/p11-kit/default.nix

```
@@ -61,6 +92,10 @@ stdenv.mkDerivation rec {
       PKCS#11 modules in such a way that they're discoverable.
     '';
     homepage = "https://p11-glue.github.io/p11-glue/p11-kit.html";
+    changelog = [
+      "https://raw.githubusercontent.com/p11-glue/p11-kit/${version}/NEWS"
```

This link is hopefully more stable:

```suggestion
      "https://github.com/p11-glue/p11-kit/raw/${version}/NEWS"
```

---

Changed